python/lsst/ts/criopy/m1m3/AccelerationAndVelocitiesFitter.py

```diff
@@ -0,0 +1,103 @@
+# This file is part of M1M3 SS GUI.
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org). See the COPYRIGHT file at the top - level directory
+# of this distribution for details of code ownership.
+#
+# This program is free software : you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+import numpy as np
+import pandas as pd
+from lsst.ts.criopy.M1M3FATable import FATABLE_XFA, FATABLE_YFA, FATABLE_ZFA
+
+
+class AccelerationAndVelocitiesFitter:
```

Comment:
Picky comment but why "acceleration" is in singular but "velocity" is in plural? wouldn't it make more sense to have them both match? Maybe `AccelerationAndVelocityFitter`?

Reply:
Renamed to ..Velocity..